docs/sql-programming-guide.md

```diff
@@ -79,7 +79,7 @@ The entry point into all functionality in Spark is the [`SparkSession`](api/java
 
 The entry point into all functionality in Spark is the [`SparkSession`](api/python/pyspark.sql.html#pyspark.sql.SparkSession) class. To create a basic `SparkSession`, just use `SparkSession.builder`:
 
-{% include_example init_session python/sql.py %}
+{% include_example init_session python/SparkSQLExample.py %}
```

Comment:
The original file name `sql.py` should be fine. Actually, the new name `SparkSQLExample.py` doesn't conform to Python convention. (You may check other file names under `examples/src/main/python`.)